src/app/features/rpc-transaction-request/rpc-transaction-request.layout.tsx

```diff
@@ -46,34 +46,18 @@ export function RpcTransactionRequestLayout({
             <BackgroundOverlay show={showOverlay} />
             <TransactionHeader
               title={title}
-              href="https://leather.io/guides/connect-dapps"
+              href={getHelpLinkUrl(method)}
               onPressRequestedByLink={e => {
                 e.preventDefault();
-                onClickRequestedByLink();
+                void analytics.track('user_clicked_requested_by_link', {
+                  endpoint: method,
+                });
+                focusTabAndWindow();
```

Comment:
this adds functionality directly into the layout component. Layout components shouldn't describe logic, hence the callback before